lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -229,7 +210,7 @@ class TraceElements extends Gatherer {
     for (const [nodeId, animationIds] of elementAnimations) {
       const animations = [];
       for (const {animationId, failureReasonsMask, unsupportedProperties} of animationIds) {
-        const animationName = await this.resolveAnimationName(passContext, animationId);
+        const animationName = await this.animationIdToName.get(animationId);
```

Comment:
```suggestion
        const animationName = this.animationIdToName.get(animationId);
```

*this one* you can remove ;)